src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -331,6 +333,41 @@ std::string postQueryContext
   }
 
 
+  // -----------------------------------------------------------------------------
+  //
+  // <DEBUG>
```

Comment:
I understand this block of code is only to print log traces. Is that correct?

Please add the corresponding FIXME.

Reply:
Fixed in e3543782c